docs/content/en/docs/install/upgrade.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Upgrade
+description: How to upgrade to the latest version of the Lifecycle Toolkit
+icon: concepts
+layout: quickstart
+weight: 45
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+To upgrade from the previous version of the Keptn Lifecycle Toolkit to the latest,
+run the same command sequence used to install KLT:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Use the `--set` flag or download and edit the `values.yaml` file
+to modify the configuration as discused on the
+[Install the Lifecycle Toolkit](../install/) page.
+
+## Migrate from v0.6.0 to v0.7.0
+
+Keptn Lifecycle Toolkit Version v0.7.0
+introduces the `metrics-operator`,
+which is now separate from the `lifecycle-operator`.
+Some functionality and behavior has been moved, changed, or renamed.
+
+Specifically, the `KeptnMetricsProvider` CRD replaces
+the now-deprecated `KeptnEvaluationProvider` CRD.
+Consequently, you must manually migrate the existing functionality
+to the `KeptnMetricsProvider` CRD.
+Execute the following external bash script to do this:
```

Comment:
should this be run before helm upgrade?

Reply:
Good question.  But I think we can discuss this and fix it in the next iteration, right?  I also need to create the reference page for the yaml file for the KeptnEvaluationProvider CRD and I expect cross-refs both to and from that page.